_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+## Terminal: Support for changing the font size
+
+Users can now change the font size in the emulator window on the
+terminal page.
+
+![New font size selector in the terminal](/images/terminal-font-size.png)
+
+### Machines: Allow editing disk cache mode
+
+It's now possible to edit the cache mode for existing virtual machine
+disks.  This was previously only possible when creating new disks.  In
+order to edit the cache mode, any machines using the volume need to be
+powered off.
```

Comment:
```suggestion
Cache mode is now editable for existing virtual machine images, when a machine is in a "shut off" state. Previously, it could only be set within Cockpit while creating a new image.
```

Reply:
The opinion — that passive voice should be avoided — is had by me.

Reply:
It's now possible to not say something other than "it's now possible to" all the time in the release notes.

```suggestion
Edit cache mode for existing virtual disks in this new release. Previous versions of Cockpit only allowed editing the cache mode during disk creation. To edit the cache mode, machines using the volume need to be "shut off".
```

There were a lot of issues with the existing paragraph, including "powered off", as we call it "shut off" in the UI, as does libvirt and everything that uses it.

Reply:
'machines using the volume' is not correct here. volume is reffering to libvirt's storage volume concept, however not all disks have a corresponding volume.

Reply:
What's the requirement to ensure the disk's cache mode can be edited?